src/execution/compiler/operator/analyze_translator.cpp

```diff
@@ -52,6 +51,18 @@ AnalyzeTranslator::AnalyzeTranslator(const planner::AnalyzePlanNode &plan, Compi
     aggregate_variables_.emplace_back(agg_var);
     pg_statistic_column_lookup_[col_info.column_oid_] = agg_var;
   }
+  ast::Expr *storage_interface_type = GetCodeGen()->BuiltinType(ast::BuiltinType::StorageInterface);
+  pg_statistic_updater_ = pipeline->DeclarePipelineStateEntry("storageInterface", storage_interface_type);
+}
+
+void AnalyzeTranslator::InitializePipelineState(const Pipeline &pipeline, FunctionBuilder *function) const {
+  // @storageInterfaceInit(&pipelineState.storageInterface, execCtx, table_oid, col_oids, true)
+  DeclareAndInitPgStatisticUpdater(function);
```

Comment:
Would we need to remove the call to this function and `FreePgStatisticUpdater` on lines 123 and 153?